### DISCUSSION TITLE
Random effects for state-dependent distribution parameters?

### DISCUSSION THREAD

**Discussion by pr0nica** [bmcclintock/momentuHMM#157] [Q&A] [answered]
Hello, 

I am working with accelerometry data in female reindeer and am using momentuHMM to classify two states from this data (resting/active). I am modelling the state probabilities as a function of temperature and time of day, as well as multiple individual-level covariates (ID, calf status, body mass). I have a total of 14 individuals, and they differ slightly in their acceleration data distributions. I want to include ID as a random effect in the state-dependent distribution parameters, so each state assignment is based on the individual's distribution, not the distribution at the group level. I read your paper on random effects in Methods Ecol Evol. 2021, but this seems to deal only with random effects in the stationary state and transition probabilites, not with random effects in the state-dependent distributions. Is there a way to include random effects/individual level values in the state-dependent distribution parameters, but still model the responses for all individuals in the same model (i.e. group response)?

Could potentially fitting a random effect term for ID for the state transition probabilities solve this problem instead?

Kind regards, 
Monica

**Comment by bmcclintock** [comment, marked as answer]
If there is individual variation in the state-dependent observation distributions, then a random effect for ID on the state transition probabilities will not capture it. Unfortunately, {momentuHMM} currently does not support random effects for the state-dependent observation distribution parameters. The best you could do with {momentuHMM} is include ID as a fixed effect on the observation distribution parameters or, better yet, individual-level covariates (e.g. sex, age) that help explain the individual-level variation.

**Reply by pr0nica** [reply]
Thank you for your response. Yes, it was also my understanding that it was not possible to include random effects for the state-dependent observation distribution parameters in momentuHMM. I have indeed included individual-level covariates, and compared these against models with ID as a fixed effect. Best, Monica
